gcovr/__main__.py

```diff
@@ -259,13 +260,17 @@ def main(args=None):
 def collect_coverage_from_tracefiles(covdata, options, logger):
     datafiles = set()
 
-    for trace_file in options.add_tracefile:
-        if not os.path.exists(normpath(trace_file)):
+    for trace_files_regex in options.add_tracefile:
+        trace_files = glob(trace_files_regex)
```

Comment:
Do we want to use `glob(..., recursive=True)` in order to allow `**/*.json` style patterns? I see no particularly good use case for this, but I also see no good reason to disallow recursive glob patterns.

Reply:
I can see no reason to disallow `recursive=True` so I have added this option in next version of patch. Thanks for hint!